openshift/templates/tools/pipeline.yaml

```diff
@@ -40,14 +39,11 @@ objects:
   status:
     lastVersion: 0
 parameters:
-- name: "APP_NAME"
-  displayName: "Application Name"
-  description: "Name of the application this pipeline will build and deploy."
-  required: true
-- name: "COMP_NAME"
-  displayName: "Component Name"
-  description: "Name of the application component this pipeline will build and deploy."
+- name: "NAME"
+  displayName: "Pipeline Name (without -pipeline suffix)"
+  description: "Name of the pipeline, e.g cicd, promote-to-test, sonarqube. A -pipeline suffix will be appended to the name provided."
   required: true
+  value: cicd
```

Comment:
I’d be tempted to no default the name so that whomever uses the template provides one.

Reply:
Fixed